src/classes/application.js

```diff
@@ -160,6 +161,11 @@ module.exports = class Application extends Base {
     const app = express()
     _app.set(this, app)
 
+    log.trace('Ash server loading cors middleware')
+    if (config.cors !== false) {
```

Comment:
See my comment in https://github.com/ash-framework/ash/pull/4 - I think you need to check this value better.

10/10 if my config is:

```js
module.exports = function (environment) {
  const ENV = {
    port: 3010
  }

  return ENV
}
```

then this block becomes

```js
if (config.cors !== false) { // undefined !== false -> true
  app.use(cors(Object.assign({}, undefined)))
}
```